analysis/measures_definition_pf_codes_conditions.py

```diff
@@ -145,6 +145,24 @@
             intervals=months(monthly_intervals).starting_on(start_date),
         )
 
+denominator_uncomplicated_uti = (age>=16) & (age<=64) & (patients.sex.is_in(["female"]))
+denominator_shingles = age>=18
+denominator_impetigo = age>=1
+denominator_infected_insect_bites = age>=1
+denominator_acute_sore_throat = age>=5
+denominator_acute_sinusitis = age>=12
+denominator_acute_otitis_media = (age>=1) & (age<=17)
+
+denominators = {
```

Comment:
great implementation, I'd just make it a bit clearer that these are the denominators for the pharmacy first conditions

```suggestion
pf_condition_denominators = {
```